### PR TITLE
Meeting 06.md

### PR DESCRIPTION
**Pull request by Santhosh1692**

### CODE REVIEW

File: Merge/Meeting 06.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+  - Polish merge/beacon-chain.md, ethereum/eth2.0-specs#2472
+  - Add randao to execution payload, ethereum/eth2.0-specs#2479
+- Spec discussion
+  - Location of execution-layer specs
+- Open discussions
+
+# Intro
+**Mikhail Kalinin**
+* Let's just get started yeah welcome to the Merge Implementers' Call #6 today is going to be pretty straightforward I guess so we'll start from implementation updates as usual there is yeah there is update from my side i'm working on the transition process implementation attacker because the pr has been merged into the into this big repo so I feel like i'll finish this implementation like t the end of this week or early next week and we'll try to.
+* Yeah, the next challenge will be to set up the proofwork chain locally around the beacon chain, do all the steps that we did on the mainnet already, and finish it with the merge transition process, so probably I'll just you yeah yeah likely reuse the scripts that we have from after rayonism, which is great, and yeah we'll use the Gary PR, which already contains everything.
```
Comment (poojaranjan) [inline Merge/Meeting 06.md:22]:
```suggestion
* Yeah, the next challenge will be to set up the proof of work chain locally around the beacon chain, do all the steps that we did on the mainnet already, and finish it with the merge transition process, so probably I'll just you yeah yeah likely reuse the scripts that we have from after rayonism, which is great, and yeah we'll use the Gary PR, which already contains everything.
```

File: Merge/Meeting 06.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+* The transition logic and in gaf so we'll see this is my update does anyone else have an implementation update yep I believe it makes sense everyone is working on the corresponding hard forks so okay cool any questions here.
+* Okay, let's go on to the research updates. We have a couple of things to highlight and possibly discuss, and we have Justin on the line I guess 
+
+# [Research Discussion](https://youtu.be/b5gh0Mw2oPU&t=70s)
+
+**Danny**
+* I can provide some context. I'm talking about 24/72. Justin is known for going through polishing and merging making sure that the specs conform in terms of naming conventions and structure and all that kind of stuff, so he's done that on the recent merge specs has been about a bunch of review I just did it's now passing ci and and has incorporated the feedback so I gave it a plus one this morning if anyone wants to take a look at it.
```
Comment (poojaranjan) [inline Merge/Meeting 06.md:29]:
```suggestion
* I can provide some context. I'm talking about PR [2472](ethereum/eth2.0-specs#2472). Justin is known for going through polishing and merging making sure that the specs conform in terms of naming conventions and structure and all that kind of stuff, so he's done that on the recent merge specs has been about a bunch of reviews I just did it's now passing ci and and has incorporated the feedback so I gave it a plus one this morning if anyone wants to take a look at it.
```

File: Merge/Meeting 06.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+* Yeah, the execution layer gets context and directives, so I don't think it's a major concern, but I'm not in charge of the program on that side.
+
+**Mikhail Kalinin**
+* yeah, this is the question, particularly for net client monitors, how tough is it to adjust the.
+
+**Danny**
+* Well, that will have to be adjusted again. It could be something like if post-merge difficulty equals one um or if post-merge difficulty equals this value that has been passed on.
+
+**Micah Zoltu**
+* Would appreciate hearing from Tommaso or Ryan.
```
Comment (poojaranjan) [inline Merge/Meeting 06.md:52]:
```suggestion
* Would appreciate hearing from Tomasz or Ryan.
```

File: Merge/Meeting 06.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+**Mikhail Kalinin**
+* I believe we require a timeline with checkboxes.
+
+**Tim Beiko**
+* Yeah, I'm arguing for checkboxes to avoid the timeline, and I believe we did this fairly well with 1559 where we have this checklist because there will be increasing pressure at the worst time when stuff is like 50 ready and people will start asking you know when merge and being able to say well look you know here are like 10 things we still need to figure out.
+* You know, both for us, I think first and foremost, but also for the community, there's value in seeing like, oh, the consensus changes are done but sync is broken or json rpc is broken or whatever right yeah, I definitely would not put dates on that docent and kind of use it as a shield against having to provide dates.
+
+**Mikhail Kalinin**
+* okay see yeah okay as for the eiep process, I don't think it makes much sense to put every different part of the execution client change into a separate dip probably we can use the approach that was taken in the click eip which just describes everything in one docent what do you think I mean once we figure out the synth process once we figure out yeah seeing process definitely will be.
```
Comment (poojaranjan) [inline Merge/Meeting 06.md:217]:
```suggestion
* okay see yeah okay as for the eip process, I don't think it makes much sense to put every different part of the execution client change into a separate dip probably we can use the approach that was taken in the click eip which just describes everything in one docent what do you think I mean once we figure out the synth process once we figure out yeah seeing process definitely will be.
```

File: Merge/Meeting 06.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+**Karim T**  
+**Kristof Gazso**  
+**Adrian Sutton**  
+**Bhargavasomu**  
+**Tim Beiko**  
+**Trenton Van Epps**  
+**Mamy**  
+**Tomasz Stanczak**  
+**Ansgar Dietrichs**  
+
```
Comment (poojaranjan) [inline Merge/Meeting 06.md:323]:
```suggestion
## Next Call - July 01, 2021 at 1300 UTC
```

**Review by poojaranjan — COMMENTED** [review]
Thanks for submitting the pull request with notes, @Santhosh1692.
A few typos need to be fixed. Would be great to have the _Summary_ section with Action items & Decisions (if any).
Overall looks good to me.

**Comment by poojaranjan** [comment]
@Santhosh1692 please add commit updating the README file too. thanks!

**Comment by Santhosh1692** [comment]
@poojaranjan Sure, will take care of this

**Comment by Santhosh1692** [comment]
@poojaranjan The comments have been resolved. It is now ready for review and merging.

**Comment by poojaranjan** [comment]
@Santhosh1692 Action/Decision items are unclear. Also, update the [README file](https://github.com/ethereum/pm/tree/master/Merge). Thanks!

**Comment by Santhosh1692** [comment]
@poojaranjan The summary and readme have been updated. I couldn't find the Twitter link for this call, so I left it blank.

**Pull request merged**